src/app/search/page.module.css

```
@@ -0,0 +1,20 @@
+.page {
+  display: flex;
+  /* grid-template-columns: 1fr 4fr; */
+  flex-direction: column;
+  align-items: start;
+  gap: 1rem;
+
+  margin-block-end: 2rem;
+
+  .filters {
+    display: flex;
+    gap: 1rem;
+    flex-direction: column;
+  }
+}
+.secondContainer {
```

بین Ruleها فاصله بذارید

```suggestion
}

.secondContainer {
```